weaviate/connect/connection.py

```diff
@@ -36,6 +39,7 @@ def __init__(
         proxies: Union[dict, str, None],
         trust_env: bool,
         additional_headers: Optional[Dict[str, Any]],
+        wait_for_weaviate: Optional[int] = 30,
```

Comment:
I would not use a default value for methods/functions that are not exposed to users. Since the user provides this value in `Client` class I think all the other class methods/ functions should not have an init value. This also allows for a better testing in case a value is not passed.